renovate.json

```
@@ -65,5 +58,6 @@
 	"semanticCommits": true,
 	"semanticCommitType": "chore",
 	"masterIssue": true,
-	"masterIssueTitle": "Renovate Dependency Updates"
+	"masterIssueTitle": "Renovate Dependency Updates",
+	"reviewers": [ "team:@Automattic/team-calypso" ]
```

I wonder if the platform team would be a more suitable one in this case?

```suggestion
	"reviewers": [ "team:@Automattic/team-calypso-platform" ]
```

---

@griffbrad @scinos, what do you think? I set it to calypso because I wasn't sure if things like Typescript would fall under the frameworks squad

---

I don't object to keeping it to the entire team, however, my inclination towards platform one is led by my expectation that any dependency upgrades are handled by the platform team by default. Of course, the platform squad is then expected to ask for reviews from the framework squad if in doubt 😉

---

That makes sense to me! It does seem like platform would be responsible in almost all cases, and then we can pass it along easily if not